Library/Formula/websocketd.rb

```diff
@@ -0,0 +1,27 @@
+require 'formula'
+
+class Websocketd < Formula
+  homepage 'http://websocketd.com'
+  url 'https://github.com/joewalnes/websocketd/archive/v0.2.10.tar.gz'
+  sha1 '4387c39aaf77e1bb5b6991358f02611a7b5dac1e'
+
+  depends_on 'go' => :build
+
+  def install
+    ENV['GOPATH'] = buildpath
+
+    system 'mkdir', '-p', 'src/github.com/joewalnes'
+    system 'ln', '-s', '../../../', 'src/github.com/joewalnes/websocketd'
+
+    system 'go', 'get', './src/github.com/joewalnes/websocketd'
+    system 'go', 'build', '-ldflags', '-X main.version ' + version, '-o', 'websocketd'
+
+    bin.install 'websocketd'
+    man1.install 'release/websocketd.man' => 'websocketd.1'
+
+  end
+
+  test do
+    system 'websocketd --version; [ $? -eq 2 ]'
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!